rpm/SPECS/openresty.spec

```diff
@@ -218,6 +228,10 @@ fi
 /usr/bin/resty
 %{orprefix}/bin/resty
 
+%files devel
+%defattr(-,root,root,-)
+
+/usr/include/nginx/*.h
```

Comment:
should I grab the headers from the `%{buildroot}/nginx/objs/` folder instead? Since NGINX doesn't install any library files that I know of and only stores some at build time.